src/repository/TransactionRepository.py

```diff
@@ -1,17 +1,20 @@
 from neo4j import ManagedTransaction
-from src.models.neo4j_models import Transaction, NFT
+from src.models.neo4j_models import Transaction, NFT, Tag
+from typing import List, Tuple
+
+
 class TransactionRepository:
 
 
     # Insert NFTS 
-    def _insert_nfts(tx: ManagedTransaction, data: list[NFT]):
+    def _insert_nfts(tx: ManagedTransaction, data: List[Tuple[NFT, List[Tuple[Tag, int]]]]):
```

Comment:
Maybe we should define some custom types in src/types , to have a clean code ? just a proposal

Reply:
@brano-hozza